app/views/batch_contexts/_new_bc_form.erb

```diff
@@ -45,6 +45,36 @@
                     See <%= link_to 'https://uit.stanford.edu/accessibility/guides/pdf'%></p>
                 </div>
             </div>
+            <div class="ocr-language d-none" data-ocr-target="ocrLanguageWrapper">
+                <div class="dropdown w-100 mb-3 d-inline-block" data-action="click@window->ocr#clickOutside">
+                    <div data-ocr-target="ocrDropdown">
+                        <div>Content language</div>
+                        <div class="border rounded ocr-search-bar d-flex">
+                            <button aria-label="toggle dropdown" aria-hidden="true" class="btn btn-link text-secondary" data-action="click->ocr#languageDropdown" aria-label="">
```

Comment:
can we get an id on this button?  it'll make it possible to test via capybara in the integration tests (I think we need to click this to get the checkboxes to show up)

Reply:
This should work find('button[aria-label=toggle dropdown]').click

Reply:
Ok, though I had to do this : `first('button[aria-label="toggle dropdown"]').click` to deal with the multiple aria-label names and also the fact that there are multiple elements with those labels.  It's also a bit more brittle than a unique ID (if a new element matching this selector gets added above this one in the form, it will break)